packages/hash/api/src/auth/index.ts

```diff
@@ -55,6 +55,14 @@ const kratosAfterRegistrationHookHandler =
       try {
         const { emails } = traits;
 
+        const hashInstanceModel = await HashInstanceModel.getHashInstanceModel(
+          graphApi,
+        );
+
+        if (!hashInstanceModel.isUserSelfRegistrationEnabled()) {
```

Comment:
Should we also be checking for `userRegistrationByInviteIsEnabled`? Or is that registration handled elsewhere?

Reply:
> Should we also be checking for userRegistrationByInviteIsEnabled?

No, that would have to be checked in some flow dedicated to invitations.

> Or is that registration handled elsewhere?

It's handled nowhere, because currently there is no org invite flow